### PR TITLE
added disability type question for demographics intake

### PR DESCRIPTION
**Pull request by olivierroyESDC**
### Description
Added the disability question for demographics page in the intake form

### Related Azure Boards Work Items
[AB#2991](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2991)
### Checklist

- [x] I have tested the changes locally
- [x] I have updated the documentation if necessary
- [x] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [ ] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [ ] I have checked that all e2e tests pass by running `npm run test:e2e`

### CODE REVIEW

File: frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx [hunk 1]
```diff
@@ -49,6 +50,23 @@ export default function DemographicsPart1() {
             />
           )}
         </div>
+
+        <div className="my-6">
+          {bornTypes.length > 0 && (
```
Comment (nicholas-ly) [inline frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx:55]:
```suggestion
          {disabilityTypes.length > 0 && (
```

Comment (olivierroyESDC) [reply frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx]: done

File: frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx [hunk 2]
```diff
@@ -49,6 +50,23 @@ export default function DemographicsPart1() {
             />
           )}
         </div>
+
+        <div className="my-6">
+          {disabilityTypes.length > 0 && (
+            <InputRadios
+              id="disability-type"
+              name="disabilityType"
+              helpMessage={t('demographics-oral-health-questions:part1.question4-note')}
```
Comment (nicholas-ly) [inline frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx:59]: Let's leverage the changes made by https://github.com/DTS-STN/canadian-dental-care-plan/pull/545 to make this the `helpMessagePrimary`.

Comment (olivierroyESDC) [reply frontend/app/routes/_public+/intake+/$id+/demographics-part1.tsx]: done

**Review by nicholas-ly — APPROVED** [review]

**Pull request merged**